### PR TITLE
Configure Debian Bullseye build

### PR DESCRIPTION
**Pull request by fklajn**
Signed-off-by: Franciszek Klajn <fklajn@opera.com>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -211,6 +211,19 @@ jobs:
 
     steps: *std_build_steps
 
+  build_bullseye_64bit_debs:
```
Comment (ripienaar) [inline .circleci/config.yml:214]: also need to include it down by `test_and_ship`

Comment (fklajn) [reply .circleci/config.yml:214]: Included, thanks.

**Comment by ripienaar** [comment]
Also by `gather_artifacts`

**Comment by ripienaar** [comment]
Also please rebase against main

**Comment by fklajn** [comment]
Added Bullseye to gather_artifacts (and to for loop in release_packagecloud) and rebased against master.

**Comment by ripienaar** [comment]
Unfortunately this is broken, you specifically removed the dh-systemd dependency as its merged in the main dhelper which does not exist on bullseye however the packaging does require dh-systemd.  So this is entirely broken.

We will need to create a bullseye specific package template that adjusts this.

https://github.com/choria-io/go-choria/blob/d2bc0b87874afd175e3a674cd14a99c02e9fb98e/packager/templates/debian/generic/control#L7

I am trying to get the release out but it wont have bullseye packages

**Comment by fklajn** [comment]
What about this? https://github.com/choria-io/go-choria/pull/1376

**Pull request merged**